src/chain/keysinterface.rs

```diff
@@ -80,6 +80,8 @@ pub trait KeysInterface: Send + Sync {
 	fn get_channel_keys(&self, inbound: bool) -> ChannelKeys;
 	/// Get a secret for construting an onion packet
 	fn get_session_key(&self) -> SecretKey;
+	/// Get a unique channel id
```

Comment:
This is a bit sparse. Specifically, should probably mention that we will use the funding txid once it exists at least.